### PR TITLE
New Subscription RP cmdlets 

### PR DESCRIPTION
**Pull request by navysingla**
## Description

Added new Subscription RP cmdlets and deprecating preview cmdlet. Design issue here - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/691

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@navysingla Please fix the compilation error.

File: src/Subscription/Subscription/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.7.4
+* Added new cmdlets `New-AzSubscriptionAlias`
```
Comment (msJinLei) [inline src/Subscription/Subscription/ChangeLog.md:23]: The description item of the change should be under upcoming release

File: src/Subscription/SubscriptionPreview/SubscriptionPreview.csproj [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (msJinLei) [inline src/Subscription/SubscriptionPreview/SubscriptionPreview.csproj:1]: If you put both preview project and GA project to master,  they will be released together.

File: src/Subscription/Subscription/Az.Subscription.psd1 [hunk 3]
```diff
@@ -72,7 +72,7 @@ NestedModules = @('Microsoft.Azure.PowerShell.Cmdlets.Subscription.dll')
 FunctionsToExport = @()
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = 'New-AzSubscription', 'Update-AzSubscription'
+CmdletsToExport = 'New-AzSubscription','Update-AzSubscription','New-AzSubscriptionAlias','Get-AzSubscriptionAlias','Remove-AzSubscriptionAlias'
```
Comment (msJinLei) [inline src/Subscription/Subscription/Az.Subscription.psd1:75]: Please submit a cmdlet review here https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues?q=is%3Aissue+Subscription for new cmdlet

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/Subscription/Subscription/Az.Subscription.psd1 (hunk 3 above)
Comment (navysingla) [reply src/Subscription/Subscription/Az.Subscription.psd1:75]: Cmdlets already approved, please check here - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/691

File: src/Subscription/SubscriptionPreview/SubscriptionPreview.csproj (hunk 2 above)
Comment (navysingla) [reply src/Subscription/SubscriptionPreview/SubscriptionPreview.csproj:1]: We are adding just GA project now

File: src/Subscription/Subscription/ChangeLog.md (hunk 1 above)
Comment (navysingla) [reply src/Subscription/Subscription/ChangeLog.md]: Fixed

**Comment by navysingla** [comment]
@msJinLei Can we merge this PR?

**Review by msJinLei — APPROVED** [review]

**Comment by msJinLei** [comment]
> @msJinLei Can we merge this PR?

@navysingla  Could you refer all the new added cmdlet in the ChangeLog.md?

**Comment by navysingla** [comment]
> > @msJinLei Can we merge this PR?
> 
> @navysingla All others looks good. Could you refer all the new added cmdlet in the ChangeLog.md?

@msJinLei Added all the cmdlets in changelog, can we merge now? and what would be the new version of cmdlets?

**Review by msJinLei — APPROVED** [review]

**Pull request merged**